app/models/miq_dialog/seeding.rb

```diff
@@ -0,0 +1,70 @@
+class MiqDialog
+  module Seeding
+    extend ActiveSupport::Concern
+
+    DIALOG_DIR = Rails.root.join("product/dialogs/miq_dialogs")
+
+    module ClassMethods
+      def seed
+        transaction do
```

Comment:
@jrafanie Also note that I missed this transaction bit in the reports PR, so I'll send a separate PR to update that

Reply:
ding ding ding.  Good call.